eslint.config.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const neo = require('neostandard')
+
+module.exports = [
+  ...neo({
+    ts: true,
+    ignores: [
+      'lib/configValidator.js',
+      'lib/error-serializer.js',
+      'test/same-shape.test.js',
+    ],
+  }),
+  {
+    ignores: [
+      'test/same-shape.test.js',
+    ],
+  },
```

Comment:
@mcollina This restores comma-dangle to what it was in `standard@17`. If the change in neostandard is controversial, then we should revert back to what `standard@17` had, to honor the _descriptive, not prescriptive_ mantra (https://github.com/neostandard/neostandard/issues/7)

A third option would be to simply relax `neostandard` itself so that it accepts both trailing and non-trailing and leave it out of `neostandard` to enforce any specific part

```suggestion
  {
    ignores: [
      'lib/configValidator.js',
      'lib/error-serializer.js',
      'test/same-shape.test.js',
    ],
    rules: {
      '@stylistic/comma-dangle': ['error', {
        arrays: 'never',
        objects: 'never',
        imports: 'never',
        exports: 'never',
        functions: 'never',
      }],
    }
  },
```